src/results/wind.jl

```diff
@@ -56,50 +56,43 @@ function add_wind_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
 	r["year_one_om_cost_before_tax"] = round(value(per_unit_size_om) / (p.pwf_om * p.third_party_factor), digits=0)
 
 	if !isempty(p.s.storage.types.elec)
-		prod_to_storage = @expression(m, [ts in p.time_steps],
+		WindToStorage = @expression(m, [ts in p.time_steps],
 			sum(m[:dvProductionToStorage][b, t, ts] for b in p.s.storage.types.elec))
 	else
-		prod_to_storage = zeros(length(p.time_steps))
+		WindToStorage = zeros(length(p.time_steps))
 	end
-	r["year_one_to_battery_series_kw"] = round.(value.(prod_to_storage), digits=3)
+	r["year_one_to_battery_series_kw"] = round.(value.(WindToStorage), digits=3)
 
-    r["year_one_to_grid_series_kw"] = zeros(size(r["year_one_to_battery_series_kw"]))
     r["average_annual_energy_exported_kwh"] = 0.0
     if !isempty(p.s.electric_tariff.export_bins)
-        wind_to_grid = @expression(m, [ts in p.time_steps],
+        WindToGrid = @expression(m, [ts in p.time_steps],
                 sum(m[:dvProductionToGrid][t, u, ts] for u in p.export_bins_by_tech[t]))
-        r["year_one_to_grid_series_kw"] = round.(value.(wind_to_grid), digits=3).data
-
+        r["year_one_to_grid_series_kw"] = round.(value.(WindToGrid), digits=3).data
         r["average_annual_energy_exported_kwh"] = round(
             sum(r["year_one_to_grid_series_kw"]) * p.hours_per_time_step, digits=0)
-    end
-
-	generatorToGrid = @expression(m, [ts in p.time_steps],
-		sum(m[:dvProductionToGrid][t, u, ts] for u in p.export_bins_by_tech[t])
-	)
-	r["year_one_to_grid_series_kw"] = round.(value.(generatorToGrid), digits=3)
-
-	prod_to_load = @expression(m, [ts in p.time_steps],
-		    m[:dvRatedProduction][t, ts] * p.production_factor[t, ts] * p.levelization_factor[t] -
-			prod_to_storage[ts] - generatorToGrid[ts]
+	else
+		WindToGrid = zeros(length(p.time_steps))
+	end
+	r["year_one_to_grid_series_kw"] = round.(value.(WindToGrid), digits=3)
+	
+	WindToCUR = (m[Symbol("dvCurtail"*_n)][t, ts] for ts in p.time_steps)
+    r["year_one_curtailed_production_series_kw"] = round.(value.(WindToCUR), digits=3)
+	
+	WindToLoad =(m[:dvRatedProduction][t, ts] * p.production_factor[t, ts] * p.levelization_factor[t] 
+			- r["year_one_to_battery_series_kw"][ts] 
+			- r["year_one_to_grid_series_kw"][ts] 
+			- r["year_one_curtailed_production_series_kw"][ts] for ts in p.time_steps
 	)
-	r["year_one_to_load_series_kw"] = round.(value.(prod_to_load), digits=3)
+	r["year_one_to_load_series_kw"] = round.(value.(WindToLoad), digits=3)
 
-	Year1GenProd = @expression(m,
-		p.hours_per_time_step * sum(m[:dvRatedProduction][t,ts] * p.production_factor[t, ts]
-			for ts in p.time_steps)
-	)
-	r["year_one_energy_produced_kwh"] = round(value(Year1GenProd), digits=0)
+	Year1WindProd = (sum(m[Symbol("dvRatedProduction"*_n)][t,ts] * p.production_factor[t, ts] for ts in p.time_steps) * p.hours_per_time_step)
```

Comment:
The expression `m[Symbol("dvRatedProduction"*_n)][t,ts] * p.production_factor[t, ts] for ts in p.time_steps` is in at least three places in these results. I think that we can save time in results processing (for all results) by avoiding redundant expressions and especially `JuMP.value` calls. @Bill-Becker has been looking into the time consumption in v1/2 of the API in the results processing. Bill: do you have any insights that could speed things up in the REopt.jl results? Should we reduce redundant `JuMP.value` calls and/or additional `@expression`s if we can?

Reply:
@Bill-Becker any thoughts here?

Reply:
addressed in https://github.com/NREL/REopt.jl/pull/75/commits/e64422d11776657f2c0b3b99f8956a9126050439